### PR TITLE
Release 1.35.2

### PR DESCRIPTION
**Pull request by renatho**
* Update changelog.
* Bump version.
* Bump tested version.
* Update pot file.

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ Tags: job manager, job listing, job board, job management, job lists, job list,
 Requires at least: 5.5
 Tested up to: 5.7
```
Comment (donnapep) [inline readme.txt:5]: Are we good with bumping this to 5.8?

File: readme.txt [hunk 2]
```diff
@@ -4,7 +4,7 @@ Tags: job manager, job listing, job board, job management, job lists, job list,
 Requires at least: 5.5
```
Comment (donnapep) [inline readme.txt:4]: 5.6 if we're bumping to 5.8.

File: readme.txt (hunk 1 above)
Comment (renatho) [reply readme.txt]: Good catch! Updated here: https://github.com/Automattic/WP-Job-Manager/pull/2175/commits/5f30812b27c29dcb8a68441a67fd3f70537a04c2

Yep, I tested it and it's working well.

File: readme.txt (hunk 2 above)
Comment (renatho) [reply readme.txt]: Also in https://github.com/Automattic/WP-Job-Manager/commit/5f30812b27c29dcb8a68441a67fd3f70537a04c2

**Review by donnapep — APPROVED** [review]

**Pull request merged**